app/uk/gov/hmrc/tai/connectors/CompanyCarConnector.scala

```diff
@@ -25,19 +25,23 @@ import uk.gov.hmrc.tai.model.domain.benefits.{CompanyCarBenefit, WithdrawCarAndF
 import uk.gov.hmrc.tai.model.enums.APITypes
 import uk.gov.hmrc.tai.model.tai.TaxYear
 
+import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class CompanyCarConnector @Inject()(httpHandler: HttpHandler, urls: PayeUrls)(implicit ec: ExecutionContext)
     extends CompanyCarBenefitFormatters {
 
+  def extraHeaders(hc: HeaderCarrier) = hc.withExtraHeaders("CorrelationId" -> UUID.randomUUID().toString)
```

Comment:
CompanyCarConnector does not need a CorrelationId as it's not a DES call, you can delete this.

Reply:
Removed , done